src/Microsoft.Extensions.Caching.Redis/project.json

```diff
@@ -11,10 +11,21 @@
   },
   "dependencies": {
     "Microsoft.Extensions.Caching.Abstractions": "1.0.0-*",
-    "Microsoft.Extensions.Options": "1.0.0-*",
-    "StackExchange.Redis.StrongName": "1.0.333"
+    "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {}
+    "net451": {
+      "StackExchange.Redis.StrongName": "1.0.488"
+    },
+    "dnxcore50": {
+      "dependencies": {
+        "StackExchange.Redis.StrongName": "1.1.553-alpha"
```

Comment:
I think we can do this soon - we've been running the latest in production on Stack Overflow for over a week. The wait was on some coreclr buffered types and such to land. We'll also try tracking [this issue](https://github.com/StackExchange/StackExchange.Redis/issues/337) (core only) down this week.